src/humancellatlas/data/metadata/api.py

```diff
@@ -132,12 +145,14 @@ class ProjectContact:
 
     @classmethod
     def from_json(cls, json: JSON) -> 'ProjectContact':
+        temp = json.get('project_role')
```

Comment:
Not a fan of `temp` as a name but I see there is precedent for it.

```suggestion
        project_role = json.get('project_role')
```

I like reassigning variables if there extraction goes through a progression of steps.